src/intrprtr.c

```diff
@@ -71,7 +71,7 @@
 **  the  statement  that  was  last  interpreted (which   might  have been  a
 **  return-value-statement).
 */
-Obj IntrResult;
+/* TL: Obj IntrResult; */
```

Comment:
This `TL: ` prefix is rather cryptic. How about we consistently write something like:
```
/* Obj IntrResult;  -- definition has been moved to InterpreterState */
```
or so? That would help people trying to understand the code.